### ISSUE TITLE
GameOS Ignoring "sort_title" criteria in my metadata files

### ISSUE THREAD

**Issue by turnarklight** [PlayingKarrde/gameOS#171]
I've been putting together my Pegasus Library, but I've noticed a bit of an issue with the GameOS theme in particular and how it is sorting my games. 

My pegasus.metadata files have 2 criteria for individual games, 
"game:" which shows the displayed title of the game, and;
"sort_title:" which determines how the games are sorted. 

For example, Road Rash, Road Rash II and Road Rash 3 have stupid titles that have both roman numerals and numbers, but if I make the "sort-by" titles "Road Rash 1" "Road Rash 2" and "Road Rash 3" I can make them sort in the correct order while still maintaining the proper display name.

The issue is, gameOS seems to be only taking the "game:" title into account and is ignoring my "sort_title" criteria altogether, even though on other themes they are displaying in the proper order I intended. 

Does GameOS lack this functionality, or is the theme looking for a different tag other than "sort_title" or "sorttitle" for the function I'm describing?

**Comment by turnarklight** [comment]
I seem to have tweaked it myself to get the result I wanted, so I guess case closed, but it would be pretty cool if sort_title could be implemented without needing to go through this weird backdoor method.

If anyone else runs into this issue, what I did was go into the theme.qml file, and where it says: 

// Filtering options
    property bool showFavs: false
    property var sortByFilter: ["title", "lastPlayed", "playCount", "rating"]
    property int sortByIndex: 0

I changed "title" to "sort_title."

In addition, in the "ListCollectionGames" file in the "Lists" folder, and in that file, where it says:

sourceModel: collection.games
        filters: [
            ValueFilter { roleName: "favorite"; value: true; enabled: showFavs },
            RegExpFilter { roleName: "title"; pattern: searchTerm; caseSensitivity: Qt.CaseInsensitive; enabled: searchTerm != "" },

I changed "title" to "sort_title" here as well.

I am not sure if both of these changes are why it got fixed or if only one of them was needed. I'd certainly be curious to know!
